### PR TITLE
EnergyHistogram

### LINKED ISSUE #4: Energy Histogram

### PR DESCRIPTION
**Pull request by deepanshlodhi**

### CODE REVIEW

**Comment by kuldeep0020** [comment]
@abhishek0220 , please check the code standard.

File: dashboard/services/utils.py [hunk 1]
```diff
@@ -98,3 +98,61 @@ def GroupMonthlyPowerData(samples: List[ElectricalDataSchema]) -> List[Timestamp
...
+
+    tsToSampleMap: DefaultDict[int, List[ElectricalDataSchema]] = defaultdict(list)
+
+    for sample in samples:
+        hourTS = GetRoundOffHourTS(sample.GenerationTimeStamp)
+        tsToSampleMap[hourTS].append(sample)
+
+    for ts, ts_samples in tsToSampleMap.items():
+
+        avg_power = sum([sample.EnergyConsumption for sample in ts_samples]) / len(ts_samples)
```
Comment (abhishek0220) [inline dashboard/services/utils.py]: change it to avg_energy for all the three

Comment (deepanshlodhi) [reply dashboard/services/utils.py]: done

**Review by abhishek0220 — APPROVED** [review]
Can be merged

**Pull request merged**